Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -160,12 +160,55 @@ export function extend(publicAPI: object, model: object, initialValues?: IVolume
  */
 export function newInstance(initialValues?: IVolumeMapperInitialValues): vtkVolumeMapper;
 
+/**
+ * Helper class to perform MPR.
+ */
+
+export interface vtkSliceHelper {
+  /**
+   * 
+   */
+   update(): void; 
+
+  /**
+   * Get the distance between clip planes
+   */
+   getThickness(): number;
+
+  /**
+   * Get the origin of the MPR
+   */
+   getOrigin(): Vector3;
+
+  /**
+   * Get the orientation of the MPR
+   */
+   getNormal(): Vector3;
+
+  /**
+   * Set the distance between clip planes
+   */
+   setThickness(thickness: number): void;
+
+  /**
+   * Set the origin of the MPR
+   */
+   setOrigin(origin: Vector3): void;
+
+  /**
+   * Set the orientation of the MPR
+   */
+   setNormal(origin: Vector3): void;
```

Comment:
does not return void but a boolean (true when normal is modified) (same for all setters)

parameter name is normal and not origin.